doc/changes/DM-48785.misc.rst

```diff
@@ -0,0 +1 @@
+The source code was reformatted with the new version of ``black`` so purely formatting changes will not interfere with any other future changes.
```

Comment:
No need for a new fragment for this. It's not a "user visible" change.

Reply:
Got it, will remove.